source/form.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Sedona</title>
+  </head>
+  <body>
+    <!-- Шапка сайта -->
+    <header class="hedaer">
+      <h1 class="header__title title-hidden">Sedona</h1>
```

Comment:
```suggestion
      <h1 class="header__title visually-hidden">Sedona</h1>
```
Думаю стоит сделать универсальное название класса для скрытия элементов, т.к. скрывать будем скорее всего не только заголовки